### DISCUSSION TITLE
console.log von scripts - wie bekommt man die auch im ioBroker LOG angezeigt

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2553] [General]
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/b26f35ac-d804-4255-8cb1-ee0d347c99f1)

console.log die man im scripts erzeugt werden nicht an das ioBroker LOG übertragen.

Kann man das irgendwo einstellen?

**Comment by Zefau** [comment]
https://github.com/Zefau/ioBroker.jarvis/issues/2550
